### PR TITLE
Updating NOTICE.txt

### PR DESCRIPTION
**Pull request by amyblais**
Second try.

There are also these but I'm not sure where to find the license: 

golang.org/x/crypto/acme/autocert
golang.org/x/crypto/bcrypt
golang.org/x/image/bmp
golang.org/x/net/bpf
golang.org/x/sys/unix

### CODE REVIEW

File: NOTICE.txt [hunk 1]
```diff
@@ -2788,40 +2765,6 @@ OR THE USE OR OTHER DEALINGS IN THE SOFTWARE.
 
 ---
 
-## html2text
```
[the remark this answers is not in the document]
Comment (amyblais) [reply NOTICE.txt]: It's a dup

File: NOTICE.txt [hunk 2]
```diff
@@ -3491,3 +3272,167 @@ OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
...
+* HOMEPAGE
+  * https://github.com/golang/freetype
+
+* LICENSE
+
+Use of the Freetype-Go software is subject to your choice of exactly one of
+the following two licenses:
+  * The FreeType License, which is similar to the original BSD license with
+    an advertising clause, or
+  * The GNU General Public License (GPL), version 2 or later.
```
Comment (jasonblais) [inline NOTICE.txt]: We cannot use GPL. Propose we keep the original entry of freetype and keep both homepages (.org and github)

File: NOTICE.txt [hunk 3]
```diff
@@ -685,30 +520,9 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
...
-LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT
-HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT
-LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY
-THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF
-THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
-
----
-
-This product contains a modified portion of 'manners',  a package imaging providing basic image manipulation functions (resize, rotate, flip, crop, etc.) by Grigory Dryapak.
+This product contains a modified portion of 'manners', a package imaging providing basic image manipulation functions (resize, rotate, flip, crop, etc.), built by Grigory Dryapak.
```
Comment (jasonblais) [inline NOTICE.txt]: Is it 'imaging'?

**Comment by amyblais** [comment]
I can't find golang.org/x/net/bpf and golang.org/x/sys/unix, can you give the line numbers?

File: NOTICE.txt [hunk 4]
```diff
@@ -8,7 +8,9 @@ This document includes a list of open source components used in Mattermost Serve
 
 ---
 
-This product contains a modified portion of 'jquery-dragster', a drag and drop listener
+## jquery-dragster
```
Comment (crspeller) [inline NOTICE.txt]: This isn't a golang dependency. This is for the webapp. Maybe we should review the list and match what I posted?

**Comment by amyblais** [comment]
@crspeller Removed extras that were not on your list.

**Pull request merged**